### PR TITLE
Viewport Segments: Fix mitigation by removing the focused area one

### PR DESCRIPTION
**Pull request by darktears**
and list the visibility one.

### CODE REVIEW

**Comment by darktears** [comment]
@kenchris

File: segments-explainer/SEGMENTS-EXPLAINER.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Two mitigation strategies could be added:
 
 ### Mitigation strategies
 - Secure Context: the `segments` are only available within a secure context.
-- Focused Area: `segments` are only available for each active document whose origin is the same origin-domain with the currently focused area document.
+- Visibility state: the `segments` property will return stale value and is only updated while the document's visibility state is "visible" or just changed to "visible".
```
Comment (kenchris) [inline segments-explainer/SEGMENTS-EXPLAINER.md]: may return?

Comment (kenchris) [reply segments-explainer/SEGMENTS-EXPLAINER.md]: why is the "or just changed to" needed? seems to be included in the former

File: segments-explainer/SEGMENTS-EXPLAINER.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ Two mitigation strategies could be added:
 
 ### Mitigation strategies
 - Secure Context: the `segments` are only available within a secure context.
-- Focused Area: `segments` are only available for each active document whose origin is the same origin-domain with the currently focused area document.
+- Visibility state: the `segments` property may return stale value and is only updated while the document's visibility state is "visible".
```
Comment (kenchris) [inline segments-explainer/SEGMENTS-EXPLAINER.md]: a stale value or stale values

**Review by kenchris — APPROVED** [review]

**Pull request merged**